src/schema/v2/homeView/HomeViewCard.ts

```diff
@@ -0,0 +1,40 @@
+import Image from "../image"
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { connectionWithCursorInfo } from "../fields/pagination"
+
+export const HomeViewCardType = new GraphQLObjectType<any, ResolverContext>({
+  name: "HomeViewCard",
```

Comment:
Yeah, that's a good thought. We could potentially define a singular-version of the section type defined here which exposes a single "card".

But since the viewAll behavior for the "Galleries Near You" section navigates to a simple list of gallery partners, I wonder if instead we could define an alternate styling of the generic Gallery section type. Maybe the default styling of such a section renders the list of galleries in a rail.

```graphql
{
  homeView {
    section(id: "home-view-section-galleries-near-you") {
      __typename // "HomeViewSectionGalleries"
      component {
        title // "Galleries Near You"
        type // "TeaserCard"
        description // "Follow these local galleries for updates on artists you love."
        backgroundImageURL // "https://files.artsy.net/images/galleries_for_you.webp"
        behaviors {
          viewAll {
            buttonText // "Explore"
          }
        }
      }
    }
  }
}
```

_Note that I think the `backgroundImageURL` string attribute above could easily be upgraded to a `backgroundImage` field backed by our more robust Image type._

<img width="300" src="https://github.com/user-attachments/assets/e34f2c5f-e7a3-4e29-bdce-c68458a7c0b4" />

Reply:
Yeah in that^ case I'd be in favor of reclaiming the `HomeViewSectionGalleries` name for an _actual_ connection of galleries, which is not at all what that section/component does now. 

> Note that I think the backgroundImageURL string attribute above could easily be upgraded to a backgroundImage field backed by our more robust Image type.

Good idea — placeholder ticket: [ONYX-1339](https://artsyproduct.atlassian.net/browse/ONYX-1339)

Reply:
Agree on this, this could be re-used on Galleries components as singular Card component.